go/nbs/dynamo_manifest.go

```diff
@@ -39,13 +40,14 @@ type ddbsvc interface {
 type dynamoManifest struct {
 	table, db string
 	ddbsvc    ddbsvc
+	lock      addr
```

Comment:
Seperately, I noticed when debugging last night, it appeared as though I watched repeated openings of the same manifest which, each time resulted in a different ordering of the table files. I wondered what that was about. Not because it's wrong, just surprising.